apps/governance/src/routes/proposals/components/vote-breakdown/vote-breakdown.tsx

```diff
@@ -0,0 +1,378 @@
+          data-testid="vote-status"
+          className="flex items-center gap-1 mb-2 text-bold"
+        >
+          <span>
+            {willPass ? (
+              <Icon name="tick" size={5} className="text-vega-green" />
+            ) : (
+              <Icon name="cross" size={5} className="text-vega-pink" />
+            )}
+          </span>
+          <span>{t('currentlySetTo')} </span>
+          {willPass ? (
+            <span>
+              <span className="text-vega-green">{t('pass')}</span>
+              {isUpdateMarket && <span> {updateMarketVotePassMethod}</span>}
+            </span>
+          ) : (
+            <span className="text-vega-pink">{t('fail')}</span>
+          )}
+        </div>
+      )}
+
+      {isUpdateMarket && (
+        <div className="mb-4">
+          <h3 className={headingClasses}>{t('liquidityProviderVote')}</h3>
+          <div className={sectionWrapperClasses}>
+            <section data-testid="lp-majority-breakdown">
+              <VoteProgress
+                percentageFor={yesLPPercentage}
+                colourfulBg={true}
+                testId="lp-majority-progress"
+              >
+                <Status
+                  reached={majorityLPMet}
+                  threshold={requiredMajorityLPPercentage}
+                  text={t('majorityThreshold')}
+                  testId={
+                    majorityLPMet ? 'lp-majority-met' : 'lp-majority-not-met'
+                  }
+                />
+              </VoteProgress>
+
+              <div className={progressDetailsClasses}>
+                <div className="flex items-center gap-1">
+                  <span>{t('liquidityProviderVotesFor')}:</span>
+                  <Tooltip
+                    description={formatNumber(
+                      yesEquityLikeShareWeight,
+                      defaultDP
+                    )}
+                  >
+                    <button>
+                      {yesEquityLikeShareWeight
+                        .dividedBy(toBigNum(10 ** 6, 0))
+                        .toFixed(1)}
+                      M
+                    </button>
+                  </Tooltip>
+                  <span>
+                    (
+                    <Tooltip
+                      description={
+                        <span>{yesLPPercentage.toFixed(defaultDP)}%</span>
+                      }
+                    >
+                      <button>{yesLPPercentage.toFixed(0)}%</button>
+                    </Tooltip>
+                    )
+                  </span>
+                </div>
+
+                <div className="flex items-center gap-1">
+                  <span>{t('liquidityProviderVotesAgainst')}:</span>
+                  <Tooltip
+                    description={formatNumber(
+                      noEquityLikeShareWeight,
+                      defaultDP
+                    )}
+                  >
+                    <button>
+                      {noEquityLikeShareWeight
```

Comment:
suggestion: I think this could be managed by `CompactNumber` component.

Reply:
I had a play but no immediate luck. I'll merge it like this for now but will chat with you and revisit